src/js/components/locations-configuration/LocationDetails.jsx

```diff
@@ -61,6 +61,9 @@ const FIELDS = {
     attributes: {
       withTooltip: true,
       tooltip: 'react.locationsConfiguration.locationNumber.tooltip.label',
+      tooltipUrl: 'https://openboxes.atlassian.net/wiki/spaces/OBW/pages/1291452471/Configure+Organizations+and+Locations',
```

Comment:
These attributes are a bit odd. I think it would be better to add a `tootlipHtml` attribute, and then assign it to html param on Tooltip. So you could have there `html={tooltipHtml ? tooltipHtml : translate(tooltip, tooltip)}`